src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue

```diff
@@ -99,13 +113,16 @@ import NcAppNavigationItem from '@nextcloud/vue/dist/Components/NcAppNavigationI
 import Participant from '../Participants/ParticipantsList/Participant/Participant.vue'
 import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
 import BreakoutRoomsEditor from '../../BreakoutRoomsEditor/BreakoutRoomsEditor.vue'
+import NcActionButton from '@nextcloud/vue/dist/Components/NcActionButton.js'
+import SendMessageDialog from '../../BreakoutRoomsEditor/SendMessageDialog.vue'
```

Comment:
All changes seem OK for me, but different approach in different components to the organisation of imports worries me a little bit.
Maybe we could move it to separate discussion?

Reply:
Yeah, we should straighten that after the RC begin, but before we split of stable26

Reply:
Yes, let's open an issue for that and decide what to do there!